.travis.yml

```diff
@@ -1,4 +1,7 @@
 language: android
+
+script: "./gradlew build javadoc"
```

Comment:
I've changed also [travis.yml](https://github.com/gotev/android-upload-service/blob/master/.travis.yml) and it now builds without "137 killed" error.